bundles/org.openhab.binding.tibber/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -122,4 +123,10 @@
 		<description>JSON array of tuples startsAt,total, e.g. {["startsAt": "2022-09-10T00:00:00+02:00", "total": 5.332},
 			{"startsAt": ...}]. See binding documantation for full example.</description>
 	</channel-type>
+	<channel-type id="today_prices" advanced="true">
+		<item-type>String</item-type>
+		<label>Prices for today as a JSON array</label>
+		<description>JSON array of tuples startsAt,total, e.g. {["startsAt": "2022-09-10T00:00:00+02:00", "total": 5.332},
+			{"startsAt": ...}]. See binding documantation for full example.</description>
```

Comment:
```suggestion
			{"startsAt": ...}]. See binding documentation for full example.</description>
```

Reply:
It's back. I see you fixed it also for tomorrow in i18n properties - please also fix it accordingly in this file.